src/structurizr/api/structurizr_client.py

```diff
@@ -206,11 +207,13 @@ def lock_workspace(self) -> bool:
         logger.debug("%r", response.json())
         response.raise_for_status()
         response = APIResponse.parse_raw(response.text)
+
         if not response.success:
-            logger.error(
+            logger.warn(
                 f"Failed to lock workspace {self.workspace_id}. {response.message}"
             )
-        return response.success
+
+        return self._paid_plan(response), response.success
```

Comment:
Return type of the method needs updating to match

Reply:
I'd also be tempted to put the success first as it's the more significant of the two values.